modules/core/src/classes/matrix4.ts

```diff
@@ -240,6 +240,34 @@ export class Matrix4 extends Matrix {
     return this.check();
   }
 
+  /**
+   * Calculates a 4x4 matrix from the given matrix3
+   * @param matrix3 
+   * @returns self
+   */
+  fromMatrix3(matrix3: Readonly<NumericArray>): this {
```

Comment:
This is fine, though it does add a little to the size of the Matrix4 class for every user.

Reply:
As you can see in the CI log, you need to run `yarn lint fix`

You are welcome to join our OpenJS slack channel, see https://openvisualization.org for the link